<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDisplaySettingDlg</class>
 <widget class="QDialog" name="QDisplaySettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDisplaySettingDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelMaxRenderSecs">
       <property name="text">
        <string>  Max draw seconds (1-99s)(Restart softwave effective): </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMaxRenderSecs">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Intelligent Wave Scale</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>When the front </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxIntelligentWaveScale">
        <property name="text">
         <string>Use Intelligent Wave Scale</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>characters of wave name same, use one wave scale.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxHashNameCharCount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxDrawMedicalECGWave">
     <property name="text">
      <string>Draw Medical ECG Wave. For 1mV sample data will draw 1cm height.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxDrawWaveScaleText">
     <property name="text">
      <string>Draw wave scale text.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>124</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
